transcom/mymove
# B 20939 int

## [B-20939](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20939)

## Summary
This task adds in the preparation date field for the SSW in the top right hand corner, and includes some other miscellaneous organization changes and test fixes to make the service easier to work with for upcoming features.

### How to test

1. Access the application as a service member.
2. Create and submit a PPM with an advance request.
3. Log in to the office app as a service counselor.
4. Edit shipment details and approve the advance.
5. Edit order details and insert missing data.
6. Submit move information (This is the action that triggers AOA Preparation date as per AC)
7. Download the AOA Packet. Ensure the preparation date is present.
8. Log back in as service member, optionally download and check again.
9. Complete the closeout process by uploading documents.
10. Log back in as Service Counselor, approve the closeout documents. (This is the action that triggers the preparation date for the SSW per AC)
11. Download the Payment Packet, ensure preparation date is present.
12. Optionally, log back in as a customer and download the payment packet, ensure preparation date is present.

### Backend


## Screenshots
<img width="553" alt="image" src="https://github.com/user-attachments/assets/46dc4964-b8be-4a2b-9bd8-17088cd9f216">
